test/model/src/include/model/driver/kv_workload.h

```diff
@@ -592,6 +592,53 @@ operator<<(std::ostream &out, const insert &op)
     return out;
 }
 
+/*
+ * get --
+ *     A representation of reading a given key. Does not state the expected value.
+ */
+struct get : public with_txn_id, public with_table_id {
+    data_value key;
```

Comment:
Nit: extra line here

Reply:
Took me a moment - you mean I should add a new line after 600 so it matches the pattern elsewhere (like the insert op I was copying actually, oops). Sure, can do.